cmd/translate/main_grpc_integration_test.go

```diff
@@ -14,26 +18,65 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-func createConnection(ctx context.Context, t *testing.T) (*grpc.ClientConn, error) {
-	t.Helper()
+var (
+	transferProtocol string // http or https
+	addr             string
+	port             string
 
-	conn, err := grpc.DialContext(
-		ctx,
-		"localhost:8080",
+	client pb.TranslateServiceClient
+)
+
+func TestMain(m *testing.M) {
```

Comment:
I once again moved `TestMain` function to gRPC integration test. 
**Reason**: The tests were failing because the server was not ready to accept requests.
Fixed with using `grpc.DialContext` with `grpc.WithBlock` which blocks execution while connection is not established